buildpipeline/pipelines.json

```diff
@@ -70,6 +70,19 @@
             "PB_BuildType": null
           }
         },
+        {
+          "Name": "DotNet-CoreClr-Trusted-FreeBSD",
+          "Parameters": {
+            "Rid": "freebsd"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "FreeBSD",
+            "Type": "build/product/",
+            "SubType": "PortableBuild",
```

Comment:
I know this is the same thing that OSX says, but it sort of confuses me since it's explicitly not portable (both passing portable=false and the makefile explicitly rejects being in portable on non-Linux OSes).